src/seeds/test/tokens.ts

```diff
@@ -0,0 +1,17 @@
+import * as Knex from "knex";
+import Promise = require("bluebird");
```

Comment:
Is it necessary to import from bluebird here?

Reply:
Unfortunately, yes:

```
src/seeds/test/tokens.ts(5,5): error TS2322: Type 'Bluebird<any>' is not assignable to type 'Promise<any>'.
  Property '[Symbol.toStringTag]' is missing in type 'Bluebird<any>'.
```

Reply:
As explained here:

https://github.com/petkaantonov/bluebird/issues/1277

Bluebird is not compatible with ES6 / TypeScript2 Promises because of this missing field. It's an open issue that hasn't been merged into mainline yet.